apps/web/src/components/(marketing)/faster-smarter-beautiful-bento.tsx

```diff
@@ -16,6 +16,21 @@ export const FasterSmarterBeautifulBento = ({
   className,
   ...props
 }: FasterSmarterBeautifulBentoProps) => {
+  const featureCards = [
+    {
+      title: 'Beautiful.',
+      description:
+        'Because signing should be celebrated. That’s why we care about the smallest detail in our product.',
+      image: cardBeautifulFigure,
+    },
+    {
+      title: 'Smart.',
+      description:
+        'Our custom templates come with smart rules that can help you save time and energy.',
+      image: cardSmartFigure,
+    },
+  ];
```

Comment:
Prefer to keep this in JSX since it's more natural to edit even if it is a little duplication.

Reply:
Gotcha, reverting this change